doc-content/enterprise-only/admin-and-config/maven-external-configure-proc.adoc

```diff
@@ -3,6 +3,9 @@
 
 You can configure {CENTRAL} and {KIE_SERVER} to use an external Maven repository, such as Nexus or Artifactory, instead of the built-in repository. This enables {CENTRAL} and {KIE_SERVER} to access and download artifacts that are maintained in the external Maven repository.
 
+[IMPORTANT]
+Artifacts in repositories do not receive automated security patches as Maven requires artifacts to be immutable. As a result, artifacts that are missing patches for known security flaws will remain in the repository to avoid breaking builds that depend on them. Patched artifacts have their version number incremented. For more information, see https://access.redhat.com/maven-repository[JBoss Enterprise Maven Respository].
```

Comment:
because Maven requires (do not us as to mean because)
The version numbers of patched artifacts are incremented.

Reply:
Agreed. Is it ok to make larger changes to the wording when it is based on source such as https://access.redhat.com/maven-repository?

Reply:
Note: Discussed and agreed to use text without modification and to link to it.